src/facet/inspection/_shap.py

```diff
@@ -670,7 +677,7 @@ def _convert_raw_shap_to_df(
                 )
 
             # all good: proceed with SHAP values for class 1 (positive class):
-            raw_shap_tensors: List[np.ndarray] = raw_shap_tensors[1:]
+            raw_shap_tensors = raw_shap_tensors[1:]
```

Comment:
Because `raw_shap_tensors` is also name of the argument to this method. Repeating type signature causes mypy redefinition error that we aren't allowing.